utility/data_fetcher.go

```diff
@@ -38,6 +46,13 @@ func (d *ManualDecoder) IntPtr(base map[string]interface{}, key string, keys ...
 		if valFloat == float32(valInt) {
 			return &valInt
 		}
+	} else if valNumber, ok := val.(json.Number); ok {
```

Comment:
As this should be the default now, to you mind moving it up to the first `else if`?

Reply:
good idea, done